deepchem/models/tensorgraph/layers.py

```diff
@@ -3212,3 +3212,23 @@ def none_tensors(self):
 
   def set_tensors(self, tensor):
     self.out_tensor, self.betas = tensor
+
+
+class PassThroughLayer(Layer):
```

Comment:
We already have a class that does this.  See `LayerSplitter`.

Reply:
While similar -- LayerSplitter cannot handle the case of different sized out_tensors.  I will change AlphaShare to use PassThroughLayer and remove LayerSplitter.